### ISSUE TITLE
POK-31 - chore: Fork repo and deploy to Netlify

### ISSUE THREAD

**Issue by wijohnst** [wijohnst/vanilla-pokedex#31]
### TL/DR
 In order to make changes to the code and deploy it, you will need to fork the repo to your own GitHub profile and then deploy it using [Netlify](https://www.netlify.com/?utm_source=google&utm_medium=paid_search&utm_campaign=12755510784&adgroup=118788138897&utm_term=netlify&utm_content=kwd-309804753741&creative=514583565825&device=c&matchtype=b&location=9014966&gclid=Cj0KCQiAjbagBhD3ARIsANRrqEtWOiw1K-55u80ntihM9vS0NTOIOdtFBxoOlRKeq9QKp1AeTEIJdSwaAifWEALw_wcB).

### Task

1. Fork the "vanilla-pokedex" repo to your own GitHub profile by clicking the "Fork" button in the top right corner of the repo page.
2. Clone your fork of the repo to your local machine using the command line:

	`git clone https://github.com/YOUR-USERNAME/vanilla-pokedex.git`

3. Install the necessary dependencies by running the following command in the project directory:

	`npm install`

4. Build the project to check for any build errors or warnings by running the following command:

	`npm run build`

5. Ensure that there are no build errors or warnings. If there are any errors, fix them before proceeding.
6. Create a new site in Netlify by linking your GitHub account and selecting the "vanilla-pokedex" repo from your list of repositories.
7. Set the build command to `npm run build`  and the publish directory to `dist/` .
8. Deploy the site by clicking the `Deploy Site`  button in Netlify.
9. Once the site is deployed, make some changes to the code to customize the app or make an improvement
10. Test the site thoroughly to ensure that it is free from runtime errors. If there are any errors, fix them before proceeding.
11. Commit your changes and push them to your fork of the repo
12. Merge your changes back into the original "vanilla-pokedex" repo. The changes, once merged in `main` should result in your forked version of the app being redeployed and the app being updated with your changes.

### Acceptance Criteria

- The site is successfully deployed via Netlify.
- Customizations have been made to the code and are visible on the deployed site.
- The site is free from all build and runtime errors.
- The changes have been committed and pushed to a fork of the "vanilla-pokedex" repo.
- A pull request has been created to merge the changes back into the original repo.

**Comment by mathiuskitchens** [comment]
All set!
